.github/workflows/web-test-unit-tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Unit tests for web code
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/web-test.yml"
+      - "web-admin/**"
+      - "web-auth/**"
+      - "web-common/**"
+      - "web-local/**"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Filter modified codepaths
+        uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            admin:
+              - ".github/workflows/web-test.yml"
+              - "admin/**"
+              - "cli/**"
+              - "runtime/**"
```

Comment:
Do the unit tests hit the runtime at all? We don't need to run them when `cli` or `runtime` code is changed, right?